dita/regions/europe/france/unit_a.dita

```diff
@@ -100,6 +101,16 @@
                 <xref href="fr_cooling_units.dita" format="dita"/>
             </rellink>
         </links>
+        -->
+        
+        <related-links scope="local" format="dita">
+            <link href="fr_power_units.dita">
+                <linktext>Fr power unit</linktext>
```

Comment:
can we remove the `linktext`, so the name comes from the document?

(it's a bit less for the author to edit/maintain)